sdk/python/feast/feature_store.py

```diff
@@ -278,12 +285,12 @@ def get_historical_features(
             >>> fs = FeatureStore(config=RepoConfig(provider="gcp"))
             >>> retrieval_job = fs.get_historical_features(
             >>>     entity_df="SELECT event_timestamp, order_id, customer_id from gcp_project.my_ds.customer_orders",
-            >>>     feature_refs=["customer:age", "customer:avg_orders_1d", "customer:avg_orders_7d"]
-            >>> )
+            >>>     feature_refs=["customer:age", "customer:avg_orders_1d", "customer:avg_orders_7d"],
+            >>>     feature_names_only=True
```

Comment:
Just thinking out loud: If we are going to make the default `True`, would it make more sense to invert the condition? `full_feature_name=False`?

Reply:
your suggestion makes sense. Seems to be more standard afaik

Reply:
I agree with you both, will change the function signatures.